packages-nextgen/kestrel_core/src/kestrel/frontend/mapping/property/stix.yaml

```diff
@@ -0,0 +1,196 @@
+autonomous-system:name: organization.name
+autonomous-system:number: organization.uid
+
+directory:path: file.path
+directory:accessed: file.accessed_time
+directory:created: file.created_time
+directory:modified: file.modified_time
+
+domain-name.value:
+  - src_endpoint.domain
+  - dst_endpoint.domain
+  - dns_query.hostname
+
+email-addr:value: user.email_addr
+email-addr:display_name: user.full_name
+
+# email-message:is_multipart
+# email-message:date
+# email-message:content_type
+email-message:from_ref.value: email.from
+email-message:sender_ref.value: email.smtp_from
+email-message:to_refs[*].value: email.to
+email-message:cc_refs[*].value: email.cc
+email-message:subject: email.subject
+# email-message:received_lines
+email-message:additional_header_fields: email.raw_header
+# email-message:body
+# email-message:body_multipart,
+# email-message:raw_email_ref
+
+# email-mime-part-type:body: file.mime_type
+# email-mime-part-type:body_raw_ref
+# email-mime-part-type:content_type
+# email-mime-part-type:content_disposition
+
+file:accessed: file.accessed_time
+file:created: file.created_time
+file:name: file.name
+file:size: file.size
+file:hashes.SHA-256: file.hashes.SHA-256
+file:hashes.SHA-1: file.hashes.SHA-1
+file:hashes.MD5: file.hashes.MD5
+file:parent_directory_ref.path: file.parent_folder
+# file:name_enc
+# file:magic_number_hex
+file:mime_type: file.mime_type
+# file:is_encrypted
+# file:encryption_algorithm
+# file:decryption_key
+# file:contains_refs
+# file:content_ref
+
+ipv4-addr:value:
+  - dst_endpoint.ip
+  - src_endpoint.ip
+  - device.ip
+# ipv4-addr.belongs_to_refs
+# ipv4-addr.resolves_to_refs
+
+ipv6-addr:value:
+  - dst_endpoint.ip
+  - src_endpoint.ip
+  - device.ip
+
+mac-addr:value: 
+  - dst_endpoint.mac
+  - src_endpoint.mac
+  - device.mac
+
+network-traffic:dst_byte_count: traffic.bytes_in
+network-traffic:dst_packets: traffic.packets_in
+network-traffic:dst_port: dst_endpoint.port
+network-traffic:dst_ref.value: dst_endpoint.ip
+
+network-traffic:protocols[*]:
+  - connection_info.protocol_num
+  - connection_info.protocol_name
+  - connection_info.protocol_ver_id
+network-traffic:src_byte_count: traffic.bytes_out
+network-traffic:src_packets: traffic.packets_out
+network-traffic:src_port: src_endpoint.port
+network-traffic:src_ref.value: src_endpoint.ip
+network_traffic:start: start_time
+network_traffic:end: end_time
+# network_traffic:is_active
+# network_traffic:ipfix
+# network_traffic:src_payload_ref
+# network_traffic:dst_payload_ref
+# network_traffic:encapsulates_refs
+# network_traffic:encapsulated_by_ref
+
+process:binary_ref.name: file.name
+process:command_line: 
+  - process.cmd_line
+  - actor.process.cmd_line
```

Comment:
This is tricky. Depending on whether we are querying `Network Activity` or `Process Activity`, `actor.process` could mean different things and it should or should not be included. Discussion needed.

Reply:
We decided to only execute process query on _Process Activity_ events and only execute network-traffic query on _Network Activity_ events (category 4 with multiple event types in OCSF). So just remove `actor.process.cmd_line` to avoid different semantics of `actor` in the two types of events.

Reply:
Removed `actor.process.cmd_line` in commit https://github.com/opencybersecurityalliance/kestrel-lang/pull/414/commits/15c9caec06163a26299565dfdf151fcf4509fcfa